sky/packages/sky/lib/src/material/dropdown.dart

```diff
@@ -133,33 +132,13 @@ class _MenuRoute extends PerformanceRoute {
   final int level;
   final int selectedIndex;
 
-  bool get ephemeral => true;
-  bool get modal => true;
   bool get opaque => false;
   Duration get transitionDuration => _kMenuDuration;
 
-  Widget build(RouteArguments args) {
-    final RenderBox renderBox = navigator.context.findRenderObject();
-    final Size navigatorSize = renderBox.size;
-    final RelativeRect menuRect = new RelativeRect.fromSize(rect, navigatorSize);
-
-    return new Positioned(
-      top: menuRect.top - (selectedIndex * rect.height),
-      right: menuRect.right - _kMenuHorizontalPadding,
-      left: menuRect.left - _kMenuHorizontalPadding,
-      child: new Focus(
-        key: new GlobalObjectKey(this),
-        autofocus: true,
-        child: new _DropdownMenu(
-          items: items,
-          selectedIndex: selectedIndex,
-          rect: rect,
-          level: level,
-          performance: performance
-        )
-      )
-    );
-  }
+  List<Widget> createWidgets() => [
+    new ModalBarrier(),
```

Comment:
This is an interesting design. (The ModalBarrier widget.)

Eventually we're going to have to have a way to disable tab-navigation to focusables earlier than the modal barrier. Not sure how this will interact with the widget here, but we'll cross that bridge when we get to it.